aspnetcore/security/authentication/certauth.md

```diff
@@ -206,3 +206,243 @@ Complete the following steps In IIS Manager:
 ### Azure and custom web proxies
 
 See the [host and deploy documentation](xref:host-and-deploy/proxy-load-balancer#certificate-forwarding) for how to configure the certificate forwarding middleware.
+
+### Use certificate authentication with the X-ARR-ClientCert header
+
+The `AddCertificateForwarding` method is used to specify:
+
+* The client header name.
+* How the certificate is to be loaded (using the `HeaderConverter` property).
+
+When sending the certificate with the `HttpClient` using the default settings, the `ClientCertificate` will always be null. The `X-ARR-ClientCert` header is used to pass the client certificate. The certificate is passed as a string to work around this problem.
```

Comment:
Too much implementation detail. Just say "In Azure Web Applications the certificate is passed as a custom header, X-ARR-ClientCert. In order to use it you must configure certificate forwarding.

Reply:
fixed